config.js

```diff
@@ -83,12 +83,20 @@ Config.prototype._seed = function _seed(seed) {
     seedOrDefault('dampScoringSuppressDuration', 60 * 60 * 1000); // 1 hr in ms
     seedOrDefault('dampScoringSuppressLimit', 5000);
 
+    seedOrDefault('lagSamplerEnabled', true);
+    seedOrDefault('lagSamplerInterval', 1000, numValidator);
+
     // Joiner config
     seedOrDefault('joinDelayMin', 100, numValidator); // ms
     seedOrDefault('joinDelayMax', 2 * 60 * 1000, numValidator); // 2 min in ms
     seedOrDefault('joinTroubleErrorEnabled', true);
     seedOrDefault('maxJoinDuration', 20 * 60 * 1000, numValidator); // 20 mins in ms
 
+    // Forwarding config
+    seedOrDefault('backpressureEnabled', true);
+    seedOrDefault('maxEventLoopLag', 1500, numValidator);
+    seedOrDefault('maxInflightRequests', 1000, numValidator);
```

Comment:
Does the back of an envelope support these defaults?  The lag makes intuitive sense to me.  I don't have a strong intuition about max inflight requests and haven't seen stats myself, though 1000 is certainly plausible.